barretenberg/ts/package.json

```diff
@@ -20,6 +20,9 @@
     "cjs-entry/",
     "README.md"
   ],
+  "engines": {
```

Comment:
Was there a strong reason to add this? I'd rather not change it unless necessary!